src/python/sdk/econia_sdk/utils/decimals.py

```diff
@@ -112,9 +181,5 @@ def get_max_quote_per_base_nominal(
     * `smallest_decimal_size_quote`: The decimal size of one tick of quote,
       as a string i.e "0.001" for one-thousandth.
     """
-    _verify_decimal_input(smallest_decimal_size_base)
-    _verify_decimal_input(smallest_decimal_size_quote)
-    lots_per_base_unit = 1 / Decimal(smallest_decimal_size_base)
-    max_ticks_per_lot = (2**32) - 1
-    max_quote_per_lot = Decimal(smallest_decimal_size_quote) * max_ticks_per_lot
-    return float(max_quote_per_lot * lots_per_base_unit)
+    _verify_decimal_input(price_precision_nominal)
+    return Decimal(price_precision_nominal) * ((2**32)-1)
```

Comment:
How about `2**32-1` as a constant like `MAX_TICKS_INTEGER` instead of a magic number?